Automattic/wp-calypso
# Store: Settings: Payments: DRY payment sections

We have a bit of duplication between the three payment sections. This DRYs that up.

To test:
* Ensure in Store > Settings > Payments you can still enable and manage on-site, off-site and offline payment methods

In support of #17271
